client/views/actions/whois.tpl

```diff
@@ -1,26 +1,35 @@
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	<i class="hostmask">({{whois.username}}@{{whois.hostname}})</i>:
-	<b>{{whois.realname}}</b>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	<i class="hostmask">({{whois.user}}@{{whois.host}})</i>:
+	<b>{{whois.real_name}}</b>
 </div>
+{{#if whois.account}}
+<div>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is logged in as <b>{{whois.account}}</b>
+</div>
+{{/if}}
 {{#if whois.channels}}
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	is on the following channels:
-	{{#each whois.channels}}
-		{{{parse this}}}
-	{{/each}}
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is on the following channels: {{{parse whois.channels}}}
 </div>
 {{/if}}
 {{#if whois.server}}
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	is connected to {{whois.server}}
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is connected to {{whois.server}} <i>({{whois.server_info}})</i>
+</div>
+{{/if}}
+{{#if whois.secure}}
+<div>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is using a secure connection
```

Comment:
`secure` here means that the user is connected to the server through TLS?

I don't mean to be the devil's advocate here, but is it a good idea to display such information? I mean, I understand that the more information the better, and that not displaying it is merely security through obscurity (therefore, not security), but that'd very easily give away information to ill-intended users.
No strong opinion, merely wondering in terms of good practice (similarly to GitHub responding a 404 on private repositories from the outside).